.Gruntfile.babel.js

```diff
@@ -318,3 +325,35 @@ function script2ify (file) {
     cb(null, buf.toString('utf8').replace(regex, replacement))
   })
 }
+
+var localeObject = {}
+function localify (file) {
+  return !/\.(vue|html|ejs)$/.test(file)
+    ? through()
+    : through(function (buf, encoding, cb) {
+      let functionEx = /\bL\(\s*['"](.*?)['"]\s*(?:,\s*['"](.*?)['"]\s*)?\)/mg
+      let markupEx = /<i18n([\u0000-\uFFFF]*?)>([\u0000-\uFFFF]*?)<\/i18n>/mg
+      let commentEx = /comment\s*=\s*["']([\u0000-\uFFFF]*?)["']/mg
```

Comment:
Per DM, I think this should be something like: `/comment\s*=\s*["']([\u0000-\uFFFF]*?)['"]\s*$/mg` <- needing that `\s*$` at the end to avoid not-full captures of strings with strings inside of them.